desktop/public_desktop/preload.js

```diff
@@ -43,11 +43,24 @@ const receiveChannels = [
 	'toggle-notification-bar',
 ];
 
+function fflagOverrides() {
+	const payload = {};
+	let fflags = process.env.WP_DESKTOP_DEBUG_FEATURES;
+	if ( fflags !== undefined ) {
+		fflags = fflags.split( ',' );
+		for ( let i = 0; i < fflags.length; i++ ) {
+			payload[ fflags[ i ] ] = true;
```

Comment:
This works well in the case we're looking at here, but what if we want to override a feature that is already set to `true` to be `false` instead?

Reply:
Very nice - good change! Code looks good. I think this is ready to land (I can't approve it myself, tho 😆 ).